compiler/src/dmd/backend/util2.d

```diff
@@ -41,7 +41,7 @@ void file_progress()
 /****************************
  * Clean up and exit program.
  */
-
+@trusted
```

Comment:
Is this required?

Reply:
This one is unrelated, the `master` branch DMD was erroring because of a `@system` call in `@safe` code